google/resource_compute_instance_template.go

```diff
@@ -262,8 +262,8 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 						"automatic_restart": &schema.Schema{
 							Type:     schema.TypeBool,
 							Optional: true,
-							Default:  true,
 							ForceNew: true,
+							Computed: true,
```

Comment:
To my understanding, `Computed` means that it can't be set and can only be read, but `Computed` and `Optional` means that it acts like it is `Computed` until a value is set in config, at which point it is treated as `Optional`.